--- a/meson.build	2023-04-29 01:27:49.458891521 +0530
+++ b/meson.build	2023-04-29 01:29:26.928590743 +0530
@@ -34,14 +34,6 @@
 # Python 3 required modules
 python3_required_modules = ['distro', 'peewee', 'mutagen', 'gi']
 
-foreach p : python3_required_modules
-  # Source: https://docs.python.org/3/library/importlib.html#checking-if-a-module-can-be-imported
-  script = 'import importlib.util; import sys; exit(1) if importlib.util.find_spec(\''+ p +'\') is None else exit(0)'
-  if run_command(python_bin, '-c', script).returncode() != 0
-    error('Required Python3 module \'' + p + '\' not found')
-  endif
-endforeach
-
 LIBEXEC_DIR = join_paths(get_option('prefix'), 'libexec')
 DATA_DIR = join_paths(get_option('prefix'), get_option('datadir'), meson.project_name())
 
@@ -97,4 +89,4 @@
     meson.add_install_script('meson/post_install_legacy.py')
 endif
 
-meson.add_install_script('meson/post_install_desktop_database.py')
\ No newline at end of file
+meson.add_install_script('meson/post_install_desktop_database.py')
